plugins/opsgenie/plugin.spec.yaml

```diff
@@ -0,0 +1,277 @@
+        type: "[]string"
+        required: false
+        example: '["Restart", "AnExampleAction"]'
+      tags:
+        title: Tags
+        description: Tags of the alert
+        type: "[]string"
+        required: false
+        example: '["OverwriteQuietHours","Critical"]'
+      details:
+        title: Details
+        description: Map of key-value pairs to use as custom properties of the alert
+        type: object
+        required: false
+        example: '{"key1":"value1","key2":"value2"}'
+      entity:
+        title: Entity
+        description: Entity field of the alert that is generally used to specify which domain an alert is related to
+        type: string
+        required: false
+        example: "An example entity"
+      source:
+        title: Source
+        description: Source field of the alert. Default value is IP address of the incoming request
+        type: string
+        required: false
+        example: "192.168.0.1"
+      priority:
+        title: Priority
+        description: Priority level of the alert. Possible values are P1, P2, P3, P4 and P5. Default value is P3
+        type: string
+        enum:
+          - ""
+          - "P1"
+          - "P2"
+          - "P3"
+          - "P4"
+          - "P5"
+        default: "P3"
+        required: false
+        example: "P1"
+      user:
+        title: User
+        description: Display name of the request owner
+        type: string
+        required: false
+        example: "ExampleName"
+      note:
+        title: Note
+        description: Additional note that will be added while creating the alert
+        type: string
+        required: false
+        example: "Example additional note"
+    output:
+      result:
+        title: Result
+        description: Result message from API
+        type: string
+        required: false
+      took:
+        title: Took
+        description: Time took to execute API
+        type: float
+        required: false
+      requestId:
+        title: Request ID
+        description: ID of a executed API request
+        type: string
+        required: false
+  get_alert:
+    title: Get Alert
+    description: Retrieve alert from OpsGenie
+    input:
+      identifier:
+        title: Alert ID, Tiny ID or Alias
+        description: Identifier of the alert
+        type: string
+        required: true
+        example: "8418d193-2dab-4490-b331-8c02cdd196b7"
+      identifierType:
+        title: Identifier Type
+        description: Type of the identifier that is provided as an in-line parameter. Possible values are ID, tiny ID and alias. Default value is ID
+        type: string
+        enum:
+          - ""
+          - "ID"
+          - "tiny"
+          - "alias"
+        default: "ID"
+        required: false
+        example: "ID"
+    output:
+      data:
+        title: Data
+        description: Data that contains JSON response
+        type: object
```

Comment:
Does this response have any schema associated with it? Or is it completely free-form? Generic objects are too difficult to use, and should be avoided when possible.

Reply:
Should we unpacked `data` then and make example schema for it?